taskana-adapter-camunda-system-connector/src/main/java/pro/taskana/adapter/systemconnector/camunda/api/impl/CamundaSystemConnectorImpl.java

```diff
@@ -18,28 +19,36 @@
     static final String URL_GET_CAMUNDA_TASKS = "/task/";
     static final String URL_GET_CAMUNDA_HISTORIC_TASKS = "/history/task/";
     static final String URL_GET_CAMUNDA_VARIABLES = "/variables/";
+    static final String URL_OUTBOX_REST_PATH = "/rest/outbox";
+
+    static final String URL_GET_CAMUNDA_START_EVENTS = "/getCreateEvents?startedAfter=";
```

Comment:
URL_GET_CAMUNDA_START_EVENTS made me think of actual start events in a camunda process instance. Meanwhile this URL is used to receive create Events started after a certain time. -> URL_GET_CAMUNDA_CREATE_EVENTS_STARTED_AFTER

Reply:
accepted